app/src/pages/organization/projectTeamPage/membersListTable/membersListTable.jsx

```diff
@@ -0,0 +1,174 @@
+export const MembersListTableWrapped = ({ members }) => {
+  const { formatMessage } = useIntl();
+  const activeProjectKey = useSelector(activeProjectKeySelector);
+  const { organizationSlug, projectSlug } = useSelector(urlOrganizationAndProjectSelector);
+  const { userRole: role, organizationRole, projectRole } = useSelector(userRolesSelector);
+  const hasPermission = role === ADMINISTRATOR || organizationRole === MANAGER;
+  const isViewer = projectRole === VIEWER;
+
+  const getPermission = (memberRole, memberOrganizationRole, memberProjectRole) => {
+    if (memberRole === ADMINISTRATOR) {
+      return formatMessage(messages.adminRole);
+    }
+    if (memberOrganizationRole === MANAGER) {
+      return formatMessage(messages.managerRole);
+    }
+    if (memberProjectRole === EDITOR) {
+      return formatMessage(messages.editorRole);
+    } else {
+      return formatMessage(messages.viewerRole);
+    }
+  };
+
+  const data = useMemo(
+    () =>
+      members.map(
+        ({
+          email,
+          fullName,
+          id,
+          metadata,
+          userRole,
+          userId,
+          assignedOrganization,
+          assignedProject,
+        }) => {
+          const memberOrganizationRole = assignedOrganization?.[organizationSlug]?.organizationRole;
+          const memberProjectRole = assignedProject?.[projectSlug]?.projectRole;
+
+          return {
+            id,
+            fullName: {
+              content: fullName,
+              component: (
+                <div className={cx('member-name-column')}>
+                  <UserAvatar
+                    className={cx('custom-user-avatar')}
+                    projectKey={activeProjectKey}
+                    userId={userId}
+                  />
+                  <div className={cx('full-name')}>{fullName}</div>
+                </div>
+              ),
+            },
+            email,
+            lastLogin: {
+              content: metadata.last_login,
+              component: metadata.last_login ? (
+                <AbsRelTime startTime={metadata.last_login} customClass={cx('date')} />
+              ) : (
+                <span>n/a</span>
+              ),
+            },
+            permissions: getPermission(userRole, memberOrganizationRole, memberProjectRole),
+          };
+        },
+      ),
+    [members, activeProjectKey, organizationSlug, projectSlug],
+  );
+
+  const primaryColumn = {
+    key: 'fullName',
+    header: formatMessage(messages.name),
+  };
+
+  const fixedColumns = [];
+
+  if (hasPermission) {
+    fixedColumns.push({
+      key: 'email',
+      header: formatMessage(messages.email),
+      width: 208,
+      align: 'left',
+    });
+  }
+
+  fixedColumns.push(
+    {
+      key: 'lastLogin',
+      header: formatMessage(messages.lastLogin),
+      width: 138,
+      align: 'left',
+    },
+    {
+      key: 'permissions',
+      header: formatMessage(messages.permissions),
+      width: 114,
+      align: 'center',
```

Comment:
I believe there is left alignment here

Reply:
probably yes, but it's not obvious:
![image](https://github.com/reportportal/service-ui/assets/45054016/534a608e-c733-46d3-b5eb-5064ea5382e4)

Reply:
Done.